app/org/sagebionetworks/bridge/dynamodb/DynamoUploadSchema.java

```diff
@@ -71,7 +72,7 @@ public String getKey() throws InvalidEntityException {
      *
      * @see #getKey
      */
-    @JsonIgnore
+    @JsonProperty
```

Comment:
What's the key? Is it either a
- guid (globally unique string)
- identifier (user generated and unique in a namespace) ? If so should be named such
If something else, it sounds like it is machine generated and assigned... is that true?